README.md

```diff
@@ -137,7 +137,7 @@ func main() {
     }
 
     // Now that the sound finished playing, we can restart from the beginning (or go to any location in the sound) using seek
-    // player.(io.Seeker).Seek(0, io.SeekStart)
+    // newPos, err := player.(io.Seeker).Seek(0, io.SeekStart)
```

Comment:
```
if _, err := player.(io.Seeker).Seek(0, io.SeekStart); err != nil {
    return err
}
```
?

Reply:
ok improved it, but separate lines to make it easier for people to read.